File: Frontend/js/admin/customer.js
$.ajax({
    type: "get",
    url: "http://localhost:8080/Backend/customer",
    success: function (response) {
        response.data.forEach((element) => {
            console.log(element);
            $("#table-body").append(
                `<tr><td>${element.email}</td>
        <td>${element.address}</td><td>${element.nic}</td><td>${element.dnum}</td>
        <td>${element.contact}</td><td>${element.password}</td></tr>`
            );
        });
    },
});

$("#home").click(function () {
    window.location.replace("../../pages/admin/admin.html");
});

File: Backend/src/main/java/lk/ijse/car_rent/entity/Customer.java
package lk.ijse.car_rent.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Customer {

    @Id
    private String email;
    private String password;
    private String nic;
    private String dnum;
    private String address;
    private int contact;
}
